mimic/model/nova_objects.py

```diff
@@ -419,8 +481,8 @@ def request_creation(self, creation_http_request, creation_json,
         """
         Request that a server be created.
         """
-        behavior = metadata_to_creation_behavior(
-            creation_json.get('server', {}).get('metadata', {}))
+        metadata = creation_json.get('server', {}).get('metadata') or {}
```

Comment:
This would also seem to defeat the check for `None` in the validator. Perhaps I'm missing something...